src/pages/explorer/index.page.tsx

```diff
@@ -6,31 +6,28 @@ import {
   InputLeftElement,
   Select,
   Text,
-} from "@chakra-ui/react";
-import type { NextPage } from "next";
-import React, { useEffect, useMemo, useRef, useState } from "react";
-import { Layout } from "@/components/Layout";
-import { useQuery } from "@apollo/client";
-import {
-  GET_ALL_NOTES,
-  GET_TAGS_AND_ENTITIES,
-} from "@/services/Apollo/Queries";
-import { SearchIcon } from "@chakra-ui/icons";
-import { useSpring, a } from "react-spring";
-import Fuse from "fuse.js";
-import { filterUniqueObjects } from "@/common/utils";
-import Router from "next/router";
-import { bodyText } from "@/theme";
-import * as styles from "./styles";
+} from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Layout } from '@/components/Layout';
+import { SearchIcon } from '@chakra-ui/icons';
+import { useSpring, a } from 'react-spring';
+import Router from 'next/router';
+import * as styles from './styles';
+import { SearchOS, SearchTypes } from '@chainverse/os';
+import { EntityTable } from '@/components/Explorer/EntityTable';
```

Comment:
Noting here that the `SearchOS` and `SearchTypes` are coming from the open source `chainverse-os` package, and the actual display of the data is coming from the `chainverse-frontend` code; exactly what we discussed for this sprint.